x/leverage/keeper/borrows.go

```diff
@@ -174,3 +174,37 @@ func (k Keeper) SetBadDebtAddress(ctx sdk.Context, denom string, borrowerAddr sd
 		store.Delete(key)
 	}
 }
+
+// GetBorrowAPY returns an sdk.Dec of an borrow APY
+// returns sdk.ZeroDec if not found
+func (k Keeper) GetBorrowAPY(ctx sdk.Context, denom string) sdk.Dec {
+	store := ctx.KVStore(k.storeKey)
+	key := types.CreateBorrowAPYKey(denom)
+
+	bytesAPY := store.Get(key)
```

Comment:
typically, you just call this `bz` -- let's be consistent if possible 👍

Reply:
done at https://github.com/umee-network/umee/pull/348/commits/9c8e4de46056c1bf6370a2ce27c5652ad7a2aaa1